### PR TITLE
Sprite changes for bedsheets/signs, minor code cleaning

### PR DESCRIPTION
**Pull request by Niknakflak**
- Replaces some of the outdated bedsheet sprites with new nicer versions made by Nienhaus.

Old sprites: 
image

New sprites: 
image
- Adds the Nanotrasen Logo bedsheet.  I just took the captains bedsheet and sprited a white N on it while removing the old gold stuff.  It looks ok.
  image
- Adds a Nanotrasen Logo sign requested by okand
  image
- Cleans up the old double barsign maltese falcon sprite and code that was left over and un-used due to that functionality and sprite being moved to barsigns.dm and dmi respectively.

### CODE REVIEW

**Comment by yackemflam** [comment]
Nothing changed

**Comment by Niknakflak** [comment]
Get your eyes checked

**Comment by Boredone** [comment]
Eh, they are pretty similar. Look at the bottom portions yackem, it got changed to shading instead of having a border between it and the higher part.

**Comment by WJohn** [comment]
If you're going to be removing the border, you should make the "bottom half" a bit darker so the shape of the bedsheet is more obvious.

File: code/game/objects/structures/signs.dm [hunk 1]
```diff
@@ -201,15 +201,10 @@
 	desc = "This plaque commemorates the fall of the Atmos FEA division. For all the charred, dizzy, and brittle men who have died in its hands."
 	icon_state = "atmosplaque"
 
-/obj/structure/sign/maltesefalcon	//The sign is 64x32, so it needs two tiles. ;3
```
Comment (ChangelingRain) [inline code/game/objects/structures/signs.dm]: turns out this is used in `wildwest.dmm`

**Comment by Nienhaus** [comment]
@WJohn I did do that, very slightly. Wasn't sure if I should do more or not.

**Comment by WJohn** [comment]
If you could make those just a tad darker, that'd be great. Sorry to bother you with that, though.

**Comment by optimumtact** [comment]
You've converted the wildwest map back to dmm

**Comment by Niknakflak** [comment]
How do I fix it

**Comment by Niknakflak** [comment]
I have remapped the sign properly for wildwest.dmm twice now.  I have map merged correctly and each time it yells that I didn't delete the right side of the falcon sign that I did after I prepared the maps and placed the new path.  I get zero errors when map merging but it is still fucking up the file.  Wildwest is just simply broken at this point if it doesn't allow anyone to map merge it.  Please help.

**Comment by optimumtact** [comment]
Can you describe in detail your map merging process? along with the path to whichever bat files you're clicking on.

**Comment by Niknakflak** [comment]
I run prepare maps.bat
I then go into wildwest.dmm and put nothing into the new path when it says the old left/right sign doesn't exist to delete them.
I then place a new barsign and save the map.
I then press map-merge tgm
and enter the number for wildwest and it successfully merges.

**Comment by optimumtact** [comment]
Alright, I will try this myself this week and let you know

**Comment by Niknakflak** [comment]
soon?

**Comment by optimumtact** [comment]
soon

**Pull request merged**